moveit_commander/src/moveit_commander/planning_scene_interface.py

```diff
@@ -63,98 +62,32 @@ def __init__(self):
 
         self._pub_co = rospy.Publisher('/collision_object', CollisionObject, queue_size=100)
         self._pub_aco = rospy.Publisher('/attached_collision_object', AttachedCollisionObject, queue_size=100)
-
-    def __make_sphere(self, name, pose, radius):
-        co = CollisionObject()
-        co.operation = CollisionObject.ADD
-        co.id = name
-        co.header = pose.header
-        sphere = SolidPrimitive()
-        sphere.type = SolidPrimitive.SPHERE
-        sphere.dimensions = [radius]
-        co.primitives = [sphere]
-        co.primitive_poses = [pose.pose]
-        return co
+        self._apply_planning_scene_diff = rospy.ServiceProxy('apply_planning_scene', ApplyPlanningScene)
+        self._apply_planning_scene_diff.wait_for_service(10.0)
```

Comment:
Isn't that overly long?
I would expect 3-5 seconds is enough.

Reply:
It depends. Perhaps this could be a param of `__init__`?

Reply:
an option sounds good. I would still propose to change the default to `5.0` here.